docs/tutorials/01_introductory/viz_texture.py

```diff
@@ -0,0 +1,37 @@
+"""
+===============
+Sphere Texture
+===============
+In this tutorial, we will show how to create a sphere with a texture.
+"""
+
+import numpy as np
+from fury import window, actor, utils, primitive, io
+import itertools
+
+##############################################################################
+# Create a a scene to start.
+
+scene = window.Scene()
+
+##############################################################################
+# Load an image (png, bmp, jpeg or jpg) using``io.prim_sphere``.
```

Comment:
missing a space between `using` and `io.prim_sphere`

Reply:
Thank you @skoudoro, I will fix these errors and reupload the tutorial!